apps/site/assets/ts/stop/components/DeparturesAndMap.tsx

```diff
@@ -33,6 +35,20 @@ const DeparturesAndMap = ({
     departureSchedules: null
   });
 
+  let allPredictions: PredictionWithTimestamp[] = [];
+  each(routes, route => {
+    each([0, 1], (directionId: DirectionId) => {
+      // eslint-disable-next-line react-hooks/rules-of-hooks
```

Comment:
Only issue, have to ignore hook rule to get linting to pass.  I don't think this will be an issue with react.